### ISSUE TITLE
Make a simulation on some CPU running in ground and can communicate with current active ship.

### ISSUE THREAD

**Issue by pack7df** [KSP-KOS/KOS#2914]

**Comment by nuggreat** [comment]
This sounds like a duplicate of issue #1366
But as there is no fleshed out description I couldn't say.

**Comment by pack7df** [comment]
I have even more ideas for evolution "realism", i think it would be very easy to implement. Like add some errors on mechanical reading given the epoch (simulating the radar detection problems and inercial guide error), and to slow down the processor speed at beginning of career and growing the speed of CPU and store capacity/mass factor when you upgrade some nodes or specific hardware on the tech tree. The communication could be at light speed, but i guess it depends on connectivity manager.

**Comment by pack7df** [comment]
Yes, it's the idea. The goal is to have a way to control the ship remotely, and to make heavy calculations. Send the results to the ship, and the ship just take care of maneuvers.
